deployments/liqo_chart/templates/configmap.yaml

```diff
@@ -8,5 +8,4 @@ data:
   clusterID: "placeHolder"
   podCIDR: {{ .Values.podCIDR }}
   serviceCIDR: {{ .Values.serviceCIDR }}
-  gatewayPrivateIP: "192.168.1.1" #placeholder, needed by broadcaster. soon will be removed
   gatewayIP: "placeholder"
```

Comment:
is this still needed?

Reply:
Nope. @alacuku ?

Reply:
I think we can remove the whole configmap. The network configuration has been move to the `clusterconfigs.liqo.io` CRD.